source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -137,6 +137,12 @@ You can also use your own function to generate paths, for example if you're usin
 
 ## JavaScript
 
+Before you import JavaScript from GOV.UK Frontend, you'll need to add the following to the top of the `<body class="govuk-template__body">` section of your page template (if you're not [using our Nunjucks macros](https://frontend.design-system.service.gov.uk/use-nunjucks/)):
```

Comment:
Mind if we keep them on the v5 docs rather than linking back?

```suggestion
Before you import JavaScript from GOV.UK Frontend, you'll need to add the following to the top of the `<body class="govuk-template__body">` section of your page template (if you're not [using our Nunjucks macros](/use-nunjucks/)):
```

Reply:
Good spot! I'll replace with `/use-nunjucks/` and that's sidestep the issue 😊